projects/netgrif-components/src/lib/navigation/navigation-double-drawer/navigation-double-drawer.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="logo" [ngClass]="{'logo-closed': !configMenu2.opened}" fxLayoutAlign="start center" fxHide.lt-md>
+    <button mat-icon-button (click)="toggleMenu2()" class="mat-button">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <img *ngIf="configMenu2.opened && image !== undefined" #logo [src]="image" class="logoimg"
```

Comment:
alt? 
The required alt attribute specifies an alternate text for an image, if the image cannot be displayed.

Reply:
done